examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -1524,6 +1540,36 @@
               "maxInterval": 65344,
               "reportableChange": 0
             }
+          ],
+          "events": [
+            {
+              "name": "StartUp",
+              "code": 0,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1
+            },
+            {
+              "name": "ShutDown",
+              "code": 1,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1
+            },
+            {
+              "name": "Leave",
+              "code": 2,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1
+            },
+            {
+              "name": "ReachableChanged",
```

Comment:
Pretty sure we never emit this event in Basic Information cluster, so we should not be listing it here or in the other example apps.

Reply:
removed ReachableChanged event from all examples